velox/docs/configs.rst

```diff
@@ -87,6 +87,11 @@ Generic Configuration
      - If set to true, then during execution of tasks, the output vectors of every operator are validated for consistency.
        This is an expensive check so should only be used for debugging. It can help debug issues where malformed vector
        cause failures or crashes by helping identify which operator is generating them.
+   * - enable_operator_buffer_cache
+     - bool
+     - true
+     - Whether to enable caches in operators. If set to false, optimizations including VectorPool in operators and
```

Comment:
```
If set to true, optimizations ... are enabled, otherwise disabled.
```

Reply:
VectorPool is used only in expression evaluation. evalWithMemo is used only in expression evaluation. Expression evaluation is used both in FilterProject and TableScan. Consider making the flag specific to expression evaluation.